<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameOSDPosition</class>
 <widget class="QWidget" name="FrameOSDPosition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="FrameTextInput" name="inputLatitude" native="true"/>
        </item>
        <item>
         <widget class="FrameTextInput" name="inputLongitude" native="true"/>
        </item>
        <item>
         <widget class="FrameOSDMode" name="mode" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>109</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameOSDMode</class>
   <extends>QWidget</extends>
   <header>src/view/panel/osd/frame_osd_mode.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrameTextInput</class>
   <extends>QWidget</extends>
   <header>src/view/shared/frame_text_input.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
